nym-registration-client/src/lp_client/nested_session.rs

```diff
@@ -32,6 +33,15 @@ use nym_wireguard_types::PeerPublicKey;
 use std::net::IpAddr;
 use std::sync::Arc;
 
+/// Create explicit bincode options for consistent serialization across versions.
+///
+/// Using explicit options future-proofs against bincode 1.x/2.x default changes.
+fn bincode_options() -> impl Options {
```

Comment:
And it'd be amazing to extend it to other bincode instances in LP, for example inside `LpMessage::encode_content`